docs/architecture/adr/ADR044-security-improvements.md

```diff
@@ -0,0 +1,113 @@
+# ADR044: Security improvements
+
+## Status
+
+Pending
+
+## Context
+
+Currently there are no restrictions on what options a service team
+puts into pod specifications.  This means that a pod in a tenant
+namespace could escalate its privileges to access resources that it
+should not have access to.
+
+Examples include:
+
+* host namespaces
```

Comment:
To what does this refer exactly?

Reply:
https://en.wikipedia.org/wiki/Linux_namespaces